packages/makeup-listbox/src/index.js

```diff
@@ -118,7 +118,7 @@ export default class {
     }
 
     get items() {
-        return this._listboxEl.querySelectorAll('[role=option]');
+        return this._listboxEl.querySelectorAll('[role=option]:not([hidden])');
```

Comment:
These items already exist inside of the active descendent, I think there shouldn't be an additional query selector.

Reply:
I'm not 100% sure about this yet, but you could be right. Same goes for makeup-menu. I'll know for sure when we start plugging in disabled and aria-disabled.